packages/pnpm/bin/pnpm.cjs

```diff
@@ -12,6 +12,7 @@ ${COMPATIBILITY_PAGE}`)
   process.exit(1)
 }
 
+global['startedAt'] = Date.now()
```

Comment:
```suggestion
```

In `packages/pnpm/src/main.ts` already has:
```ts
/* eslint-disable import/first */
if (!global['startedAt']) {
  global['startedAt'] = Date.now()
}
```

Reply:
Both are needed. I want to track the very start of execution. I track it here before the huge, bundled file is loaded. 

Tracking in main.ts is needed during local development because in that case the file from dist is directly executed.